### PR TITLE
Add squad requests to seeds

### PR DESCRIPTION
**Pull request by mirterious**

### CODE REVIEW

File: priv/repo/seeds.dev.exs [hunk 1]
```diff
@@ -18,6 +19,12 @@ for _ <- (1..seeds_config[:users]), do: insert(:user)
 # create squads
 for _ <- (1..seeds_config[:squads]), do: insert(:squad)
 
+# create users with upapproved squad request
```
Comment (ARtoriouSs) [inline priv/repo/seeds.dev.exs]: Typo

File: priv/repo/seeds.dev.exs [hunk 2]
```diff
@@ -27,6 +34,17 @@ for index <- (1..seeds_config[:squads]) do
   |> Enum.each(fn user -> insert(:squad_member, user: user, squad: squad, role: :student) end)
 end
 
+# create squad requset for users with squad
+Repo.all(User)
+|> Repo.preload(squad_member: :squad)
+|> Enum.each(fn user ->
+  if squad_member = user.squad_member do
+    build(:squad_request, user: user, squad: squad_member.squad)
+    |> approved_by(squad_member)
```
Comment (ARtoriouSs) [inline priv/repo/seeds.dev.exs]: If I'm not mistaken it means that user approved his request by himself, probably here should be a squad commander

File: spec/support/factory/accounts/user_factory.ex [hunk 3]
```diff
@@ -26,6 +26,11 @@ defmodule Squadster.Support.Factory.Accounts.UserFactory do
         %{user | squad_request: squad_request}
       end
 
+      def with_squad_request_to_a_certain_squad(user, squad) do
```
Comment (ARtoriouSs) [inline spec/support/factory/accounts/user_factory.ex]: Not a big deal, but maybe lets make this function name shorter like `with_request_to_squad` or `with_squad_request_to` or something like that

**Review by ARtoriouSs — CHANGES_REQUESTED** [review]

File: spec/support/factory/accounts/user_factory.ex (hunk 3 above)
Comment (mirterious) [reply spec/support/factory/accounts/user_factory.ex]: The first one looks great

File: priv/repo/seeds.dev.exs (hunk 2 above)
Comment (mirterious) [reply priv/repo/seeds.dev.exs]: Yep, lost it while I was refactoring code

**Comment by ARtoriouSs** [comment]
Nice!

**Review by ARtoriouSs — APPROVED** [review]

**Pull request merged**